theolive/getting-started/authorization.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 1
+sidebar_label: Authorization
+pagination_label: Authorization
+pagination_next: getting-started/regions
```

Comment:
You should only need to set `pagination_next` on `index` pages, for all other pages Docusaurus should already paginate correctly. Could you try removing these and see if it still works?

Reply:
Yep, still works. Thank you :)